projects/ad7606x_fmc/zed/system_constr.xdc

```diff
@@ -32,3 +32,5 @@ set_property -dict {PACKAGE_PIN P17     IOSTANDARD LVCMOS25} [get_ports adc_os[1
 set_property -dict {PACKAGE_PIN N17     IOSTANDARD LVCMOS25} [get_ports adc_os[2]]          ; ## H16 FMC_LPC_LA11_P
 set_property -dict {PACKAGE_PIN T19     IOSTANDARD LVCMOS25} [get_ports adc_stby]           ; ## C15 FMC_LPC_LA10_N
 set_property -dict {PACKAGE_PIN R21     IOSTANDARD LVCMOS25} [get_ports adc_range]          ; ## D15 FMC_LPC_LA09_N
+set_property -dict {PACKAGE_PIN C18     IOSTANDARD LVCMOS25} [get_ports adc_serpar]         ; ## G31 FMC_LPC_LA29_N
```

Comment:
Shouldn't those be

set_property -dict {PACKAGE_PIN K19     IOSTANDARD LVCMOS25} [get_ports adc_serpar];  ## C8 FMC_LPC_LA14_P
set_property -dict {PACKAGE_PIN K20     IOSTANDARD LVCMOS25} [get_ports adc_refsel];  ## C9 FMC_LPC_LA14_N

as package pins C18 and C19 are connected to the FMC_CLK1 on the zedboard?

Reply:
Yes, that's correct (FPGA package numbers K19 and K20, for C18, C19 as FMC pin numbers). It's my fault regarding the correspondent FPGA package number of the FMC pin.